app/services/supplementary-billing/create-billing-invoice.service.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+/**
+ * Creates a billing invoice record
+ *
+ * @module CreateBillingInvoiceService
+ */
+
+const BillingInvoiceModel = require('../../models/water/billing-invoice.model.js')
+const FetchInvoiceAccountService = require('./fetch-invoice-account.service.js')
+
+/**
+ * Create the initial billing invoice for the provided charge version, billing period and billing batch prior to
+ * transactions being sent to the charging module
+ *
+ * @param {Object[]} chargeVersion An Object containing the relevant charge version
+ * @param {Object[]} billingPeriod Object that has a `startDate` and `endDate` that defines the billing period
```

Comment:
Just ensuring we are correct in the documentation. Putting brackets after the type infers we're expecting an array

```suggestion
 * @param {Object} chargeVersion An Object containing the relevant charge version
 * @param {Object} billingPeriod Object that has a `startDate` and `endDate` that defines the billing period
```

https://jsdoc.app/tags-param.html